volume/devicemapper/devicemapper.go

```diff
@@ -19,12 +21,25 @@ const (
 	DriverName = "devicemapper"
 )
 
+var (
+	// DriverOptions are the serviced device mapper options
+	DriverOptions = []string{
+		"SERVICED_DM_THINPOOLDEV",
```

Comment:
I would consider making these constant typed strings. We may need to reference them in another package, and it would be better below to use SERVICED_DM_BASESIZE rather than "SERVICED_DM_BASESIZE".